src/EventPayload.ts

```diff
@@ -12,9 +12,18 @@ export interface EventPayload {
   action?: Action;
   /** The authorization token for the request. This will be setup from the Key or Bearer in the config. */
   authorization?: string;
-  /** Whether the event is the result of bot activity. */
+  /** Whether the event is the result of bot activity.
+   *
+   * Defaults to '$auto' which indicates to
```

Comment:
I think this phrasing is a bit confusing because to someone not to familiar with the overall system `What is '$auto'? I'm only seeing that this could be a boolean, but what type is '$auto'? If it's a string, I can imagine that this conflicting information is confusing.`

If we are just going to update the documentation, then I would avoid referencing '$auto' altogether.

Reply:
I would also figure out a consistent phrase or template to use. 

Maybe we just use something like `Will default to <some value> when processed if omitted`